src/Server.js

```diff
@@ -77,7 +76,7 @@ class Server {
     configureDoublesWithExpress() {
         function fnSendDataAndStatus(data, status) {
             return (req, res) => {
-                console.log(`DOUBLER: ${req.url}`)
+                //console.log(`DOUBLER: ${req.url}`)
```

Comment:
Let's uncomment this and leave it noisy for now. We can add config/command-line options to control it later.

Reply:
Like it